server/index.js

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable no-unused-vars */
```

Comment:
ok! doing this now -- looking at it though, whatsup with creating the server ? wondering if this interferes with mongoose.connect() calls?